wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -551,6 +612,50 @@ public void removeSnapshots(String workspaceId) throws NotFoundException, Server
     }
 
     /**
+     * Removes all snapshots of all machines of specified environment of workspace.
+     *
+     * @param workspaceId
+     *         workspace id to remove environment snapshots
+     * @param envName
+     *         environment name to remove snapshots from
+     * @throws NotFoundException
+     *           when workspace with given id doesn't exists
+     * @throws ServerException
+     *          when any other error occurs
+     */
+    @VisibleForTesting
+    void removeEnvironmentSnapshots(String workspaceId, String envName) throws NotFoundException, ServerException {
```

Comment:
Since this method is used in method **removeEnvironmentSnapshotsQuietly** only please move it under that method to ease code navigation